src/pages/App/index.tsx

```diff
@@ -1,38 +1,80 @@
 /* eslint-disable react/no-unstable-nested-components */
-/* eslint-disable @typescript-eslint/no-use-before-define */
+import { animated, config, useSpring, useTrail } from '@react-spring/web';
 import i18n from 'i18next';
-import React, { Suspense, useEffect } from 'react';
+import React, { Suspense, useEffect, useState } from 'react';
 import { I18nextProvider, useTranslation } from 'react-i18next';
-import styled from 'styled-components';
 
 // components
-import AppIcon from '../../components/AppIcon';
 import Alert from '../../components/Text/Alert';
 
-// theme
-import { animation } from '../../theme';
-
 // apps
 import { loadApp } from '../../apps';
+import { AppManifest } from '../../types';
+
+type AnimatedAppTitleProps = {
+  text: string;
+};
+
+const AnimatedAppTitle: React.FC<AnimatedAppTitleProps> = ({ text }) => {
+  const [isDisplaying, setIsDisplaying] = useState(true);
+
+  const letters = text.split('');
```

Comment:
i noticed that the loading screen text is missing spaces, i know it’s unlikely to be due to this line but could you check this please?

Reply:
Yes I know, I checked and the space is included in this split... I am investigating further